### ISSUE TITLE
Donut Batch Inference result differents from Single Inference

### ISSUE THREAD

**Issue by rjac-ml** [NielsRogge/Transformers-Tutorials#323]
I have been testing doing batch inference and compare result with single inference.

So far I have notice that the pad tokens (additional 1 into the decoder_input_ids) generate the problem. I try with the decoder_attention_mask but nothing happends. Is there a way to do batch inference but removing the pad token to have the same accuracy?

**Comment by rjac-ml** [comment]
the way that I solved the issue was using 
processor.tokenizer.padding_side = 'left'

**Comment by NielsRogge** [comment]
Thanks, I've updated the notebook!

**Comment by MitchellX** [comment]
any followups?
